bucket/rtlutility.json

```
@@ -0,0 +1,49 @@
+{
+    "version": "1.0.3",
+    "description": "RTL Utility is a tool for measuring the Round Trip Latency of your Digital Audio Workstation (DAW) and audio interface",
+    "homepage": "https://oblique-audio.com/rtl-utility.php",
+    "license": "Freeware",
+    "suggest": {
+        "vcredist2015": "extras/vcredist2015",
+        "vcredist2017": "extras/vcredist2017",
+        "vcredist2019": "extras/vcredist2019"
```

```suggestion
    "suggest": {
        "Microsoft Visual C++ Redistributables": "extras/vcredist2022"
```
The latest version includes all the redistributables for versions 2015-2019 - see https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#visual-studio-2015-2017-2019-and-2022